src/app/csv-collection/csv-collection-detail/csv-collection-detail.component.ts

```diff
@@ -1,39 +1,110 @@
-import { Component, OnInit } from '@angular/core';
+import {AfterViewInit, Component, ElementRef, NgModule, OnInit, ViewChild} from '@angular/core';
 import {Job} from '../../job/job';
 import {ActivatedRoute} from '@angular/router';
-import {NgbModal} from '@ng-bootstrap/ng-bootstrap';
+import {NgbModal, NgbModule} from '@ng-bootstrap/ng-bootstrap';
 import {JobDetailComponent} from '../../job/job-detail/job-detail.component';
 import {CsvCollectionService} from '../csv-collection.service';
 import {CsvCollection} from '../csv-collection';
 import {AppConfigService} from '../../app-config.service';
 import urljoin from 'url-join';
+import {BehaviorSubject, Observable, of as observableOf, Subject} from 'rxjs';
+import * as Flow from '@flowjs/flow.js';
+import {auditTime, catchError, map, switchMap} from 'rxjs/operators';
+import {BytesPipe, NgMathPipesModule} from 'angular-pipes';
+import {InlineEditorModule} from '@qontu/ngx-inline-editor';
+import {MatPaginator} from '@angular/material';
+import {Csv} from '../csv';
 
 @Component({
   selector: 'app-csv-collection-detail',
   templateUrl: './csv-collection-detail.component.html',
   styleUrls: ['./csv-collection-detail.component.css']
 })
-export class CsvCollectionDetailComponent implements OnInit {
+
+@NgModule({
+  imports: [NgbModule, NgMathPipesModule, BytesPipe, InlineEditorModule]
+})
+export class CsvCollectionDetailComponent implements OnInit, AfterViewInit {
 
   csvCollection: CsvCollection = new CsvCollection();
   job: Job = null;
   csvCollectionId = this.route.snapshot.paramMap.get('id');
   plotsUiLink = '';
+  uploadOption = 'regular';
+  csv: Observable<Csv[]>;
+  resultsLengthCsv = 0;
+  pageSize = 5;
+
+  pageSizeOptions: number[] = [10, 25, 50, 100];
+  csvParamsChange: BehaviorSubject<{ index: number, size: number, sort: string }>;
+
+  $throttleRefresh: Subject<void> = new Subject<void>();
+  flowHolder: Flow.IFlow;
+  displayedColumnsCsv: string[] = ['fileName', 'size'];
```

Comment:
Should be `fileSize` instead of `size` (make sure to change header reference in html as well)